charts/spire/ci/external-mysql-values.yaml

```diff
@@ -0,0 +1,11 @@
+spire-server:
```

Comment:
The test covers still the same, but instead of generating couple of things I hardcoded them. 

1. to make it work without additional plumbing around chart-testing
2. to simplify the testcases and make it more transparant for contributors
3. the tests in CI are intended more for small unit testing, e.g. testing some non default values.

For more advanced tests we can still do the examples, but that will require releases of the charts as soon we break them out due to the dependencies we will then get on certain versions.

Reply:
> The test covers still the same, but instead of generating couple of things I hardcoded them.
> 
> 1. to make it work without additional plumbing around chart-testing
> 2. to simplify the testcases and make it more transparant for contributors
> 3. the tests in CI are intended more for small unit testing, e.g. testing some non default values.

Its not unit testing though, its still integration testing. And its not testing as much as before. We need to come up with a way to get those tests back in. A unit testing framework to do unit tests would be an ok way to solve it I think.

> For more advanced tests we can still do the examples, but that will require releases of the charts as soon we break them out due to the dependencies we will then get on certain versions.

Reply:
Sure, but there haven't been tests removed here which was you original comment. We are overriding the chart default with a fixed value as opposed to a generated value. That doesn't test less neither more. So imho we didn't drop any testing here.